classes/Category.php

```diff
@@ -48,6 +48,9 @@ class CategoryCore extends ObjectModel
     /** @var mixed string or array of Description */
     public $description;
 
+    /** @var mixed string or array of Description bottom */
+    public $description_bottom;
```

Comment:
```suggestion
    public $additional_description;
```

By default, it would be at the bottom, but maybe we could have a more generic naming, ideas:
`secondary_description`
`additional_description`
`extra_description`

Reply:
@Hlavtox let's go with `additional_description`

Reply:
@kpodemski Ok Krystian! :-)